### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by retep998** [cpjreynolds/rustty#9]
Is Windows support something that is planned on being supported eventually?

**Comment by cpjreynolds** [comment]
Windows support is something I've been thinking about, however it would almost definitely have to be within the context of Cygwin. The low-level parts of the library are all based on posix standards for portability, and I'm not sure they could be translated easily to Windows; furthermore, I have no experience with software development on Windows so I would be of no help for a port using native APIs.

In all, I think support for Cygwin is very possible, but native windows support is unlikely.

**Comment by retep998** [comment]
The concept of a terminal being a grid of cells is exactly how the native Windows console works. While the low level details would need to be swapped out entirely with the Windows console API, I don't see any issues with the API of this library that would hinder that. I'd be happy to assist with implementing the native Windows support if that was desired.

**Comment by droundy** [comment]
I agree that native Windows support would be great, although I also have no windows background nor a windows computer. Possibly someone could copy what is being done on rustbox or termbox-go.
